solara/website/pages/documentation/api/routing/resolve_path.py

```diff
@@ -15,21 +15,20 @@
 @solara.component
 def Page():
     route_current, routes = solara.use_route()
-    with solara.VBox() as main:
-        # solara.Warning("Note the address bar in the browser. It should change to the path of the link.")
-        solara.Markdown("*Click on one of the links below to change the route and see the url in your browser change, and match the text.*")
-        with solara.VBox():
-            for route in routes:
-                path = solara.resolve_path(route)
-                # we could have passed the route object directly to Link, but we want to show the path
-                # can also be used.
-                with solara.Link(path):
-                    current = route_current is route
-                    if current:
-                        solara.Success(f"You are at {path}")
-                    else:
-                        solara.Info(f"{route.path} will navigate to {path}")
-    return main
+
+    # solara.Warning("Note the address bar in the browser. It should change to the path of the link.")
+    solara.Markdown("*Click on one of the links below to change the route and see the url in your browser change, and match the text.*")
+    with solara.VBox():
```

Comment:
Another while we're here update:

```suggestion
    with solara.Column():
```